docs/source/torch_environment_variables.rst

```diff
@@ -26,3 +26,4 @@ If you find anything in this documentation that is missing, incorrect, or could
    miscellaneous_environment_variables
    logging
    torch_nccl_environment_variables
+   privateuse1_environment_variables
```

Comment:
Could you please move this to the miscellaneous section? I do expect all users to be interested in this env variable, not just privateuse1 devs.
https://github.com/pytorch/pytorch/blob/34e94c507ab508ab6dea61373dc93450a9618db8/docs/source/miscellaneous_environment_variables.rst#L13

Reply:
@albanD Sure, it has been moved. Please take a look again. Thanks so much for your patient review.